cached_stores.json

```diff
@@ -0,0 +1 @@
+{"ABC Stores": [{"name": "ABC Stores", "hours": "9am - 9pm", "location": "Street Level 1, near Centerstage", "logo": "https://gizmostorageprod.blob.core.windows.net/tenant-logos/1615937914061-abcstores.png"}, {"name": "ABC Stores", "hours": "10am - 8pm", "location": "Street Level 1, in the Ewa Wing", "logo": "https://gizmostorageprod.blob.core.windows.net/tenant-logos/1615937946329-abcstores.png"}], "Apple": [{"name": "Apple", "hours": "10am - 8pm", "location": "Mall Level 2, near Macy's", "logo": "https://placewise.imgix.net/images/api/stores/10327.svg"}]}
```

Comment:
This appears to be an incomplete cache; should it include everything from the website? If its just recent queries, should it be added to `.gitignore`?

Reply:
I didn't get that. I thought this file will be being filling by stores' info during parsing mall in users' requests.